layer.md

```diff
@@ -181,7 +181,7 @@ In the absence of any [whiteout](#whiteouts) files in a layer changeset, the arc
 This section covers applying an entry in a layer changeset, if the file path already exists.
 
 If the file path is a directory, then the existing path just has it's attribute set from the layer changeset for that filepath.
-If the file path is any other file type (regular file, FIFO, etc), then the:
+If the file path is any other file type (regular file, FIFO, etc), then:
 * file path is unlinked (See [`unlink(2)`](http://linux.die.net/man/2/unlink))
```

Comment:
+1 to removing “the”, but we should also rephrase this to an imperative:

> unlink the file path (…)

But this section is getting overhauled in #317, so it may not be worth the trouble of fixing it here ;).